### PR TITLE
fix (engine): misc

### PR DESCRIPTION
**Pull request by yesnault**

### CODE REVIEW

File: vendor/github.com/fsamin/go-dump/encoder.go [hunk 1]
```diff
@@ -231,7 +231,9 @@ func (e *Encoder) ToStringMap(i interface{}) (res map[string]string, err error)
 	}
 	res = map[string]string{}
 	for k, v := range ires {
-		res[k] = fmt.Sprintf("%v", v)
```
Comment (bnjjj) [inline vendor/github.com/fsamin/go-dump/encoder.go]: does it need a PR on go-dump ?

Comment (yesnault) [reply vendor/github.com/fsamin/go-dump/encoder.go]: yes, doing

Comment (yesnault) [reply vendor/github.com/fsamin/go-dump/encoder.go]: PR: https://github.com/fsamin/go-dump/pull/14

Comment (fsamin) [reply vendor/github.com/fsamin/go-dump/encoder.go]: merged, you can update the vendor

Comment (yesnault) [reply vendor/github.com/fsamin/go-dump/encoder.go]: done

**Pull request merged**